### PR TITLE
Show first 500 entries in drop down

### PR DESCRIPTION
**Pull request by h-kataria**
Show filter and show entries from all the entries that were received from backend API. We're continuing to show 500 entries only to avoid performance issues while rendering drop-down with 5000 entries.

https://issues.redhat.com/browse/AA-1554

Backend PR https://gitlab.cee.redhat.com/automation-analytics/automation-analytics-backend/-/merge_requests/678

### CODE REVIEW

File: src/Components/Toolbar/Groups/ToolbarInput/Select.tsx [hunk 1]
```diff
@@ -76,9 +80,11 @@ const Select: FunctionComponent<Props> = ({
...
     if (textInput === '') return renderValues(selectOptions);
     return renderValues(
-      selectOptions.filter(({ value }) =>
-        value.toString().toLowerCase().includes(textInput.toLowerCase())
-      )
+      selectOptionsMasterCopy
+        .filter(({ value }) =>
+          value.toString().toLowerCase().includes(textInput.toLowerCase())
+        )
+        .slice(0, 50)
```
Comment (h-kataria) [inline src/Components/Toolbar/Groups/ToolbarInput/Select.tsx:87]: @skochay330 does this need to be 500?

Comment (AparnaKarve) [reply src/Components/Toolbar/Groups/ToolbarInput/Select.tsx:87]: @h-kataria When we filter for a specific string, we do not want to return results more than 50.

If the result is not in the first 50 items, then thats the cue for the user to refine their search further.

Comment (AparnaKarve) [reply src/Components/Toolbar/Groups/ToolbarInput/Select.tsx:87]: Usually with real data (where templates aren't named like in our test data, for e.g. - template_1, template_2, template_abc), there is a high likelihood that the filter results would be available in the first 50 items.

Comment (skochay330) [reply src/Components/Toolbar/Groups/ToolbarInput/Select.tsx:87]: @AparnaKarve ^^

**Review by skochay330 — APPROVED** [review]

**Pull request merged**